src/core/builder/brownie.ts

```diff
@@ -0,0 +1,68 @@
+import { logger } from "../helpers/logger";
+import { CompileOptions } from "../interfaces/Builder";
+import { ContractPayload } from "../interfaces/ContractPayload";
+import { BaseBuilder } from "./builder-base";
+import { execSync } from "child_process";
+import { cosmiconfigSync } from "cosmiconfig";
+import { existsSync, readFileSync, rmdirSync } from "fs";
+import { basename, join } from "path";
+
+export class BrownieBuilder extends BaseBuilder {
+  public async compile(options: CompileOptions): Promise<{
+    contracts: ContractPayload[];
+  }> {
+    const explorer = cosmiconfigSync("brownie");
+
+    const loadedConfig = explorer.load(
+      join(options.projectPath, "brownie-config.yaml"),
+    );
+
+    const buildPath = join(
+      options.projectPath,
+      loadedConfig?.config?.project_structure?.build || "./build",
+    );
+
+    if (options.clean) {
+      logger.info("Cleaning build directory");
+      existsSync(buildPath) && rmdirSync(buildPath, { recursive: true });
+    }
+
+    logger.info("Compiling...");
+    execSync("brownie compile");
+
+    const contractsPath = join(buildPath, "contracts/");
```

Comment:
is it safe to hardcode`contracts/` here?

Reply:
yes, it's always relative to the build path